src/components/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FooterContainer } from './styled-components/FooterStyles';
+
+const Contact = () => {
+  return (
+    <footer>
+      <FooterContainer>
+        <div className="contact-container">
+          <h2>CONTACT</h2>
+          <div>
+            <p>Antonella Cardozo</p>
+            <p>+46 73 963 36 03</p>
+            <p>sylcardozo.sc@gmail.com</p>
```

Comment:
would be nice with a link to the actual emailadress here. :) try the attribute "mailto" if you want to.